### PR TITLE
Fix: added env variable for updater

### PR DESCRIPTION
**Pull request by egor0798**

### CODE REVIEW

File: src/main/index.ts [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+import * as process from 'process';
 
 import { MainWindow } from './main';
 import { makeAppWithSingleInstanceLock } from './factories/instance';
 import { makeAppSetup } from './factories/setup';
 
-makeAppWithSingleInstanceLock(async () => {
-  app.commandLine.appendSwitch('autoplay-policy', 'no-user-gesture-required');
+const setupAutoUpdate = () => {
+  console.log('process.env.BUILD_SOURCE', process.env.BUILD_SOURCE);
```
Comment (Asmadek) [inline src/main/index.ts:10]: Remove console log or make it more meaningful

**Review by Asmadek — APPROVED** [review]

File: src/main/index.ts [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+import * as process from 'process';
 
 import { MainWindow } from './main';
 import { makeAppWithSingleInstanceLock } from './factories/instance';
 import { makeAppSetup } from './factories/setup';
 
-makeAppWithSingleInstanceLock(async () => {
-  app.commandLine.appendSwitch('autoplay-policy', 'no-user-gesture-required');
+const setupAutoUpdate = () => {
+  if (process.env.BUILD_SOURCE !== 'github') {
```
Comment (tuul-wq) [inline src/main/index.ts:10]: We usually make fast return this way 😉

```suggestion
  if (process.env.BUILD_SOURCE !== 'github') return;
```

**Review by tuul-wq — APPROVED** [review]
Approve with 1 comment

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -8,6 +8,7 @@ on:
 env:
   CSC_FOR_PULL_REQUEST: true
   CI: true
+  BUILD_SOURCE: 'github'
```
[the remark this answers is not in the document]
Comment (egor0798) [reply .github/workflows/release.yaml:11]: yes, that's enough. It's added here cause it shoud be set only with CI

**Review by stepanLav — APPROVED** [review]

**Pull request merged**